src/Workspaces/Core/Portable/Workspace/IWorkspaceConfigurationService.cs

```diff
@@ -22,7 +22,6 @@ internal interface IWorkspaceConfigurationService : IWorkspaceService
     [DataContract]
     internal readonly record struct WorkspaceConfigurationOptions(
         [property: DataMember(Order = 0)] StorageDatabase CacheStorage = StorageDatabase.SQLite,
-        [property: DataMember(Order = 1)] bool DisableRecoverableTrees = false,
         [property: DataMember(Order = 2)] bool DisableProjectCacheService = false,
```

Comment:
Incorrect order values.

Reply:
You can convert the primary constructor params to init-properties, which don't need to specify ordering.